.gitlab/build-deb-rpm.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+if [ -n "$CI_COMMIT_TAG" ] && [ -z "$RUBY_PACKAGE_VERSION" ]; then
+  RUBY_PACKAGE_VERSION=${CI_COMMIT_TAG##v}
+fi
+
+source common_build_functions.sh
+
+# Not sure how it is called
+fpm_wrapper "datadog-apm-library-ruby" "$RUBY_PACKAGE_VERSION" \
```

Comment:
Where does the `fpm_wrapper` command come from?
I ask because googling this gave me like no results: https://www.google.com/search?q=%22fpm_wrapper%22

Reply:
It comes from: https://github.com/DataDog/auto_inject/blob/9e0dc0f689b5acb49a1ffee53081abe994754421/packaging/common_build_functions.sh#L7